src/main/java/fi/metatavu/kuntaapi/server/integrations/GenericHttpClient.java

```diff
@@ -443,12 +444,18 @@ private String parameterToString(Object value) {
         return handleErrorResponse(statusCode, message);
     }
   }
-
-  @SuppressWarnings("unchecked")
+  
+  @SuppressWarnings({"squid:MissingDeprecatedCheck"})
+  @Deprecated
   private <T> Response<T> handleOkResponse(HttpResponse httpResponse, int statusCode, String message, TypeReference<T> typeReference) throws IOException {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)